audit_exceptions/versioned_dependencies_conflicts_allowlist.json

```diff
@@ -1,5 +1,6 @@
 [
   "couchdb-lucene",
+  "dasht",
```

Comment:
`dasht` is just shell scripts that run commands like `wget`, `w3m`, and `socat` (each of which has an OpenSSL dependency). I don't expect the mixed OpenSSL versions to cause any issue.

Reply:
This might cause some pkg-config issue in https://github.com/Homebrew/homebrew-core/pull/121566

```
bash-3.2$ pkg-config --cflags grpc++
Package openssl was not found in the pkg-config search path.
Perhaps you should add the directory containing `openssl.pc'
to the PKG_CONFIG_PATH environment variable
Package 'openssl', required by 'grpc', not found
```

Reply:
Unrelated. That's happening because OpenSSL is keg-only.

Reply:
👍